src/slider/styles/slider.m.css

```diff
@@ -31,11 +41,14 @@
 	height: 2em;
 	left: 50%;
 	margin-left: -1em;
-	position: absolute;
 	top: -0.625em;
 	width: 2em;
```

Comment:
Should dimensions be under the fixed class? The size of the track and thumb seems like one of the main things someone might want to theme.